lib/components/atoms/Label/Label.js

```diff
@@ -9,9 +9,17 @@ type Props = {
   elementLocator?: string,
   htmlFor: string,
   children: any,
+  showLabelBeforeInput: Boolean,
```

Comment:
This is a wrong change, please revert it

Reply:
This has been reverted.